.circleci/config.yml

```diff
@@ -3,23 +3,42 @@ version: 2.1
 workflows:
   test:
     jobs:
-      - build-run-linux
+    - build-run-linux:
+        context: hello-world-demos
```

Comment:
This CircleCI context provides the environment variables used on lines 40-41, so we no longer need to set variables on the individual CircleCI hello projects.